src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -2819,7 +2819,7 @@ private void DoWriteError(object obj)
         /// but the command failure will ultimately be
         /// <see cref="System.Management.Automation.ActionPreferenceStopException"/>,
         /// </remarks>
-        internal void _WriteErrorSkipAllowCheck(ErrorRecord errorRecord, ActionPreference? actionPreference = null, bool isNativeError = false)
+        internal void _WriteErrorSkipAllowCheck(ErrorRecord errorRecord, ActionPreference? actionPreference = null, bool isNativeStdErrCall = false)
```

Comment:
The change is not related to the PR. Please revert.

Reply:
Actually it's directly related to it. We now call this API from multiple native calls, but that parameter was only added for a specific scenario (the stderr scenario).

The actual issue is that we'll likely be removing it soon as the experimental feature that disables it is stabilised.